### PR TITLE
add remaining files in learn section

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/learn/account-disabling.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ locking and disabling. Follow the below steps to do this configuration.
         This will remove the account disabling option from all tenants.
 
         ``` java
```
Comment (YvonneW) [inline en/docs/learn/account-disabling.md]: The protocol should be toml—correct in other relevant places.

File: en/docs/learn/account-disabling.md [hunk 2]
```diff
@@ -113,12 +113,11 @@ steps below.
 2.  Restart the Server.
 
     !!! tip
```
Comment (YvonneW) [inline en/docs/learn/account-disabling.md]: This is an info box.

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 3]
```diff
@@ -123,12 +121,12 @@ which is `         carbon.super        ` .
     <p class="admonition-title">Note</p>
     <p>Alternatively, you can configure the expiry time in the <code>                 identity.xml                </code> configuration file.</p>
```
Comment (Buddhimah) [inline en/docs/learn/self-registration-and-account-confirmation.md]: change this to deployment.toml

File: en/docs/learn/account-disabling.md (hunk 1 above)
Comment (gomathyk) [reply en/docs/learn/account-disabling.md]: modified

File: en/docs/learn/account-disabling.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/learn/account-disabling.md]: modified

**Pull request merged**